pandas/io/sql.py

```diff
@@ -170,8 +172,49 @@ def read_sql_table(
     coerce_float=True,
     parse_dates=None,
     columns=None,
-    chunksize=None,
-):
+    chunksize: None = None,
+) -> DataFrame:
+    ...
+
+
+@overload
+def read_sql_table(
+    table_name,
+    con,
+    schema=None,
+    index_col=None,
+    coerce_float=True,
+    parse_dates=None,
+    columns=None,
+    chunksize: int = 1,
+) -> Iterator[DataFrame]:
+    ...
+
+
+@overload
```

Comment:
Hmm I don't think an overload should match the overloaded-signature should it? I think can delete?

Reply:
Yeah, that's right. I've updated the PR to remove all unneeded overloads.